pkgs/applications/networking/station/default.nix

```diff
@@ -1,13 +1,13 @@
 { appimageTools, fetchurl, lib }:
-
 let
   pname = "station";
-  version = "1.52.2";
+  version = "v2.5.0";
+  arch = "x86_64";
```

Comment:
I don't think this variable is useful because it is a string literal.

Reply:
Do you mean the arch variable?

Reply:
yes